test/e2e/landing-page.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { test, expect } from '@playwright/test';
+import { loggedInAsUser } from './utils';
+
+test.describe('landing page link', () => {
```

Comment:
```suggestion
test.describe('Landing page', () => {
```
I think this is more descriptive